projects/landing-page/src/App.tsx

```diff
@@ -89,10 +89,10 @@ const App: React.FunctionComponent = () => {
                 yarn add @substrate/substrate-connect
               </Code>
               <Code heading='index.ts'>
-                <Box>{`import { UApp } from '@substrate/connect';`}</Box>
+                <Box>{`import { Detector } from '@substrate/connect';`}</Box>
 
                 <Box mt={2}>{`// Create a new UApp with a unique name`}</Box>
-                <Box>{`const app = new UApp('burnr-wallet');`}</Box>
+                <Box>{`const app = new Detector('burnr-wallet');`}</Box>
                 <Box>{`const westend = app.detect('westend');`}</Box>
```

Comment:
Also these needs to be `await`

Reply:
There's quite a few missing `await`s actually.  Will fix

Reply:
Fixed